fahclient-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+# assumes https://github.com/NVIDIA/k8s-device-plugin or similar is deployed
+# otherwise mount in /dev/nvidia* manually (probably other steps too)
+apiVersion: apps/v1
+kind: Deployment
+metadata: 
+  name: fahclient-deployment
+  labels:
+    app: fahclient
+spec:
+  replicas: 6
+  selector:
+    matchLabels:
+      app: fahclient
+  template:
+    metadata:
+      labels:
+        app: fahclient
+    spec:
+      containers:
+      - name: fahclient
+        image: quay.io/nibalizer/fahclient:latest
```

Comment:
Can you update the image to point to the one for this project?

Reply:
can do :)